### DISCUSSION TITLE
Does it work with linux?

### DISCUSSION THREAD

**Discussion by ash-hashtag** [T-Troll/alienfx-tools#180] [Q&A]
Does the sdk depend on OS depenedent libraries? Or can we compile it directly to linux?

**Comment by T-Troll** [comment]
Yes.

Light SDK use Windows common USB HID interface.
You need to rewrite 3 methods, replacing calls from Windows to Linux API.

Additionally, data storage in Mapping class use registry now. You need to alter Save/Load methods to use file or something else.

The rest is not platform-depended, but may need some additional define(s) (like `#define WORD unsigned short`)
